core/silkworm/execution/processor.cpp

```diff
@@ -66,9 +67,12 @@ ValidationResult ExecutionProcessor::validate_transaction(const Transaction& txn
     return ValidationResult::kOk;
 }
 
-Receipt ExecutionProcessor::execute_transaction(const Transaction& txn) noexcept {
+void ExecutionProcessor::execute_transaction(const Transaction& txn, Receipt& receipt) noexcept {
     assert(validate_transaction(txn) == ValidationResult::kOk);
```

Comment:
Assert vould be voided in build release ... SILKWORM_ASSERT or runtime error ?

Reply:
`validate_transaction` is already called by `execute_block_no_post_validation` prior to `execute_transaction`. `execute_transaction` on its own is called only by tests; execution stage always goes through `execute_block_no_post_validation`. I don't want to use `SILKWORM_ASSERT` here since that would mean calling `validate_transaction` twice in production.

Reply:
Ok